fairing/deployers/serving/serving.py

```diff
@@ -41,7 +42,7 @@ def deploy(self, pod_spec):
         self.service = v1_api.create_namespaced_service(self.namespace, self.service_spec)
 
         logger.warn("Endpoint {} launched.".format(self.deployment.metadata.name))
-        url = self.backend.get_service_external_endpoint(self.deployment.metadata.name, self.deployment.metadata.namespace, self.deployment.metadata.labels)
+        url = self.backend.get_service_external_endpoint(self.deployment.metadata.name, self.deployment.metadata.namespace, self.labels)
```

Comment:
@r2d4 we needed to change this to self.labels because the deployment had no labels attached and we didn't see any method in Python API for DeploymnetSpec to attach labels.

Reply:
Sorry this needs to have 

self.service.metadata.name, self.service.metadata.namespace, self.service.metadata.labels

instead

Reply:
Ack.